test/test/cstack/CstackTests.java

```diff
@@ -41,4 +42,10 @@ public void vmStructs(TestProcess p) throws Exception {
         assert out.contains("call_stub");
         assert out.contains("JavaMain");
     }
+
+    @Test(mainClass = CpuBurner.class, os = Os.LINUX, jvmArgs = "-Xint", jvm = Jvm.HOTSPOT)
+    public void interpretedOnlyModeHasLowUnknownRatio(TestProcess p) throws Exception {
```

Comment:
No story-long method names, please. Test method names can be specified in a command line to run only selected tests; it's inconvenient to type such a name. If you want to clarify what the test is doing, add a comment instead.